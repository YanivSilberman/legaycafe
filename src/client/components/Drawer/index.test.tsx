import * as React from 'react';
import * as ReactShallowRenderer from 'react-test-renderer/shallow'
import Component from './index';

describe('Drawer component test',() => {
  it('component loads', () => {
    const renderer = ReactShallowRenderer.createRenderer();

    const User = {
      _id:"id",
      avatar:"",
      firstName:"",
      lastName:""
    }

    expect(renderer.render(
      <Component
        allUsers={[]}
        {...User}
      />
    )).toMatchSnapshot()
  })
});
