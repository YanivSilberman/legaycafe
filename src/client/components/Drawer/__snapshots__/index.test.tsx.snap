// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Drawer component test component loads 1`] = `
<withApollo(ChatDrawer)
  _id="id"
  allUsers={Array []}
  avatar=""
  classes={
    Object {
      "avatars": "withApollo(ChatDrawer)-avatars-4",
      "button": "withApollo(ChatDrawer)-button-6",
      "buttonContainer": "withApollo(ChatDrawer)-buttonContainer-5",
      "drawer": "withApollo(ChatDrawer)-drawer-1",
      "drawerPaper": "withApollo(ChatDrawer)-drawerPaper-2",
      "mainAvatar": "withApollo(ChatDrawer)-mainAvatar-3",
    }
  }
  firstName=""
  lastName=""
/>
`;
